example/main.go

```diff
@@ -27,6 +29,10 @@ func main() {
 	fmt.Println(flag.Args())
 
 	p := js.Global().Get("document").Call("createElement", "p")
-	p.Set("innerText", "Hello, World!")
+	p.Set("innerText", strings.Join([]string{
+		"Hello, World!",
+		fmt.Sprintf("args=%q", os.Args),
+		fmt.Sprintf("env=%q", os.Environ()),
+	}, "\n"))
```

Comment:
```go
	p := js.Global().Get("document").Call("createElement", "p")
	p.Set("innerText", "Hello, World!")
	js.Global().Get("document").Get("body").Call("appendChild", p)

	pre := js.Global().Get("document").Call("createElement", "pre")
	pre.Set("innerText", fmt.Sprintf("args: %q\nevn: %q", os.Args, os.Environ()))
	js.Global().Get("document").Get("body").Call("appendChild", pre)
```

Reply:
Hmm, that's better, but the `<pre>` doesn't wrap at all, which is pretty unwieldy with a lot of environment variables

Reply:
Then what about

```go
pre.Get("style").Set("whiteSpace", "pre-wrap")
```

?

Reply:
That's exactly what I was about to suggest, lol!